docs/tutorials/10_Lattice.ipynb

```diff
@@ -0,0 +1,943 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lattice"
```

Comment:
Is there some more descriptive title (few more words rather than just one) that can improve on this. At a minimum I think it should be 'Lattices' plural with a s. Lattice singular does not feel right.

Now I see this tutorial shows to the point of creating second q ops. I will make a separate comment based on this and looking at the code, but I think we should show its actual usage with an algorithm to compute something so people have an example of how it would be used to do that.